10-std.tex

```diff
@@ -539,3 +539,145 @@ \section{Input/Output}
 
 \section{Sys/sys}
 \label{std-sys}
+
+\section{Remoting}
+\label{std-remoting}
+
+Haxe remoting is a way to talk between different platforms. With Haxe remoting we can communicate transparently, send data and call methods, between server and client side.
+
+\subsection{Remoting Connection}
+\label{std-remoting-connection}
+
+In order to use remoting, there must be a connection established first. There are two kinds of Haxe Remoting connections: 
+\begin{description}
+	\item[\expr{haxe.remoting.Connection}] is used for \emph{synchronous connections}, where the results can be directly obtained when calling a method. 
+	\item[\expr{haxe.remoting.AsyncConnection}] is used for \emph{asynchronous connections}, where the results are events that will happen later in the execution process.
+\end{description}
+
+\paragraph{Start a connection}
+Haxe remoting can be used to communicate across different platforms. There are some target-specific constructors with different purposes to set up a connection:
+
+\begin{description}
+	\item[All targets:]
+		\begin{description}
+			\item[\expr{HttpAsyncConnection.urlConnect(url:String)}]  
+				Returns an asynchronous connection to the given URL that can be a Haxe/Server application. 
+		\end{description}
+		
+	\item[Flash:]
+		\begin{description}
+			\item[\expr{ExternalConnection.jsConnect(name:String, ctx:Context)}]  
+				Allows a connection to the local JavaScript Haxe code. The JS Haxe code must be compiled with the class ExternalConnection included. This is only working with Flash Player 8 and higher.
+			\item[\expr{AMFConnection.urlConnect(url:String)} and \expr{AMFConnection.connect( cnx : NetConnection )}]  
+				Allows a connection to an \href{http://en.wikipedia.org/wiki/Action_Message_Format}{AMF Remoting server} such as \href{http://www.adobe.com/products/adobe-media-server-family.html}{Flash Media Server} or \href{http://www.silexlabs.org/amfphp/}{AMFPHP}.
+			\item[\expr{SocketConnection.create(sock:flash.XMLSocket)}]  
+				Allows remoting communications over an \type{XMLSocket}
+			\item[\expr{LocalConnection.connect(name:String)}]  
+				Allows remoting communications over a \href{http://api.haxe.org/haxe/remoting/LocalConnection.html}{Flash LocalConnection}
+		\end{description}
+		
+	\item[Javascript:]
+		\begin{description}
+			\item[\expr{ExternalConnection.flashConnect(name:String, obj:String, ctx:Context)}]  
+				Allows communications a given Flash Object. Some HTML properties must be set. The Haxe Flash content must be loaded and it must include the \expr{haxe.remoting.Connection} class. This will only work with Flash 8 and higher. 
+		\end{description}
+		
+	\item[Neko:]
+		\begin{description}
+			\item[\expr{HttpConnection.urlConnect(url:String)}]  
+				Will work as the asynchronous version but in synchronous mode.
```

Comment:
consider "will work like" instead of "will work as" unless you mean that it literally will work instead of...